coffee_lib/src/repository.rs

```diff
@@ -40,5 +40,8 @@ pub trait Repository: Any {
     /// return the url of the repository.
     fn url(&self) -> URL;
 
+    /// return the git head of the repository.
+    fn git_head(&self) -> Option<String>;
+
```

Comment:
does not make sense to have this inside the repository trait

Reply:
good point. I will update it